instrumentation/restlet/restlet-core-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/restlet/v1_1/RestletInstrumentationModule.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.restlet.v1_1;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Arrays;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class RestletInstrumentationModule extends InstrumentationModule {
+
+  public RestletInstrumentationModule() {
+    super("restlet-core", "restlet-core-1.1");
```

Comment:
I named it that way because there are extensions to the Restlet library that we want to support  - e.g. Spring Extension #3662. But after giving it some thought, I guess extension instrumentation will have names like "restlet-spring-extension", so indeed naming core instrumentation just "restlet" should suffice. WDYT?

Reply:
Yeah that sounds good. In general, we aim to keep the instrumentation name as close to the library/artifact name as possible.